packages/gatsby-transformer-rehype/src/gatsby-node.js

```diff
@@ -12,18 +12,7 @@ exports.pluginOptionsSchema = function({Joi}) {
     space: Joi.string().description(`Space mode`).default(`html`),
     emitParseErrors: Joi.boolean().description(`EmitParseErrors mode`).default(`false`),
     verbose: Joi.boolean().description(`verbose mode`).default(`false`),
-    plugins:
-            Joi.subPlugins ?
-                Joi.subPlugins({entry: `index`}) :
-                Joi.array()
-                    .items(
-                        Joi.string(),
-                        Joi.object({
-                          resolve: Joi.string(),
-                          options: Joi.object({}).unknown(true),
-                        })
-                    )
-                    .default([]),
+    plugins: Joi.subPlugins(),
```

Comment:
did something break? or just cleanup?

Reply:
The validation fails if you pass `{entry: 'index'}` for ESM-bases sub-plugin (stacktrace https://ci.jenkins.io/job/Infra/job/plugin-site/job/PR-1476/9/console). Without that it should work for both Common JS and ESM.